pi-manage

```diff
@@ -586,6 +586,25 @@ def create(name, scope, action):
     return r
 
 
+@policy_manager.command
+def create_from_file(name, filename):
+    """
+    create a new policy from a JSON-file
+    """
+    import json
+
+    f = open(filename, 'r')
+    contents = f.read()
+    f.close()
+    params = json.loads(contents)
```

Comment:
I tried that first (based on the original `create`) but it threw me `ValueError: malformed string`: 
```
[splash@Macintosh:~/privacyidea]$ cat /tmp/policy.test 
{
  "action": [
    "passthru=userstore",
    "smsautosend"
  ],
  "scope": "authentication",
  "realm": [
    "localsql"
  ],
  "resolver": [
    "localusers"
  ],
  "user": "",
  "active": true,
  "check_all_resolvers": false,
  "client": "",
  "time": "",
  "adminrealm": []
}
[splash@Macintosh:~/privacyidea]$ python
Python 2.7.10 (default, Jul 30 2016, 19:40:32) 
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.34)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import ast
>>> f = open("/tmp/policy.test", 'r')
>>> contents = f.read()
>>> f.close()
>>> ast.literal_eval(contents)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ast.py", line 80, in literal_eval
    return _convert(node_or_string)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ast.py", line 63, in _convert
    in zip(node.keys, node.values))
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ast.py", line 62, in <genexpr>
    return dict((_convert(k), _convert(v)) for k, v
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ast.py", line 79, in _convert
    raise ValueError('malformed string')
ValueError: malformed string
>>> 
```

The same works with `json`:
```
>>> import json
>>> params_json = json.loads(contents)
>>> params_json
{u'realm': [u'localsql'], u'time': u'', u'active': True, u'client': u'', u'user': u'', u'resolver': [u'localusers'], u'check_all_resolvers': False, u'action': [u'passthru=userstore', u'smsautosend'], u'scope': u'authentication', u'adminrealm': []}
>>> 
```

I'll double check the documentation for `ast` if I've not missed something..